ui/app/AppLayouts/Communities/panels/MintTokensSettingsPanel.qml

```diff
@@ -221,9 +221,9 @@ StackView {
         SettingsPage {
             id: ownerTokenPage
 
-            property int chainId
-            property string accountName
-            property string accountAddress
+            property int chainId: SQUtils.ModelUtils.getByKey(root.flatNetworks, "layer", 2).chainId
```

Comment:
perhaps we need some null check here? or we can use ModelEntry and use that here instead for all 3 below.

Reply:
We can do the same as for accounts (line bellow), but basically if the model is empty it will break much earlier.